modules/ROOT/pages/_partials/commons/common-indexing.adoc

```diff
@@ -109,14 +109,201 @@ include::{root-partials}block_tabbed_code_example.adoc[]
 
 endif::is-c[]
 
+== Partial Index
+
+Couchbase Lite 3.2.2 introduces support for Partial Index - Partial Value and Partial Full-Text Indexes.
+The Partial Index can create a smaller index, potentially improving index and query performance. 
+You can use Partial Index to specify a `WHERE` clause in your index configuration.
+If a where clause is specified, the database will index a document only when the where clause condition is met. 
+
+Couchbase's query optimizer uses https://www.sqlite.org/partialindex.html[SQLite's Partial Index rules about queries using Partial Indexes] to determine whether to use partial index in the query or not.
+
+.Couchbase and SQLite's Partial Index Rules
+:!example-caption:
+====
+In general, Couchbase Lite follows the two rules, with a modification to the second rule. 
+
+Below is a summary of the two rules where:
+
+* `X` - The expression in the `WHERE` clause of a given Partial Index.
+
+* `W` - The expression in the `WHERE` clause of a given query.
+
+A query can use the Partial Index if the following two rules are satisfied:
+
+1. If `W` is AND-connected terms, and `X` is OR-connected terms and if any terms of `W` appears as a term of `X`, the partial index is allowed to be used.
+
+2. If a term in `X` is of the form `"z IS NOT MISSING"` and if a term in `W` is a comparison operator on `z` other than `"IS"`, the partial index is allowed to be used. 
+The operators include `=`, `==`, `<`, `>`, `\<=`, `>=`, `<>`, `IN`, `LIKE`, and `BETWEEN`.
+
+IMPORTANT: If `X` is in the form of `"z is NOT NULL"` or `"z is VALUED"`, the first rule must be satisfied.
+
+For example, let the partial index be `c IS NOT NULL` when creating a partial index on collection named `col1` in the default scope.
+
+Then any query that uses operators `=`, `<`, `>`, `\<=`, `>=`, `<>`, `IN`, `LIKE`, or `GLOB` on column `c` would be usable with the partial index, because those comparison operators are only true if `c` is not NULL. 
```

Comment:
Just noticed this, we don't support GLOB and BETWEEN is missing. So basically, changing BLOB to BETWEEN.

Reply:
Done